ci-operator/jobs/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master-presubmits.yaml

```diff
@@ -65,6 +65,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
you probably want `always_run: false` with a [`skip_if_only_changed` or `run_if_changed` setting][1] to avoid running this on docs bumps and the like.

[1]: https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/#tolerated-changes-to-generated-jobs

Reply:
oh, great suggestion. Thanks. I'll push over in the next minute.